src/components/thumbnail.ts

```diff
@@ -55,21 +54,36 @@ export class FrigateCardThumbnailFeatureEvent extends LitElement {
     if (this.thumbnail?.startsWith('data:')) {
       return this.thumbnail;
     }
-    return await homeAssistantSignPath(this.hass, thumbnail);
+    return new Promise((resolve, reject) => {
+      //eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      this.hass!.fetchWithAuth(thumbnail)
+        .then((response) => response.blob())
+        .then((blob) => {
+          const reader = new FileReader();
```

Comment:
This is really clever. Mind adding a comment explaining what's going on here, and why we're doing this rather than just passing a signed URL, or fetching a blob? Otherwise I can imagine some head-scratching trying to figure out "Why" :-)

Reply:
Sure, will add this now.